Fix usage of Try / Catch

marcoooo opened the pull request:
**Any pull request that does not include enough information to be reviewed in a timely manner may be closed at the maintainers' discretion**

## Description

Fix missing Try / Catch missing package error

## Use case

Package Try/Catch not present.

## Benefits

Fix package

## Possible Drawbacks

None

## Testing

- [ ] Have you added/modified unit tests to test the changes?
- [ ] If so, do the tests pass?
- [ ] Have you run the entire test suite and no regression was detected?
- [ ] TravisCI passed on your branch

Dependencies
------------

> If applicable, define what code dependencies were added and/or updated.

marcoooo commented:
I request your approval again, I added the "do not set a release-XX" dir option.

thomasmaurel commented:
> I request your approval again, I added the "do not set a release-XX" dir option.

I saw that, I guess that if you need it

marcoooo commented:
> > I request your approval again, I added the "do not set a release-XX" dir option.
> 
> I saw that, I guess that if you need it



> > I request your approval again, I added the "do not set a release-XX" dir option.
> 
> I saw that, I guess that if you need it

I needed it for covid-19 ... And btw could turn Handy for QRP as well...
